### PR TITLE
NOISSUE - enable https for bootstrap

### PR DESCRIPTION
**Pull request by mteodor**
Signed-off-by: Mirko Teodorovic <mirko.teodorovic@gmail.com>

### CODE REVIEW

File: internal/pkg/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -127,7 +129,18 @@ func Bootstrap(cfg Config, logger log.Logger, file string) error {
 }
 
 func getConfig(bsID, bsKey, bsSvrURL string) (deviceConfig, error) {
-	client := &http.Client{}
+	// Get the SystemCertPool, continue with an empty pool on error
+	rootCAs, _ := x509.SystemCertPool()
```
Comment (drasko) [inline internal/pkg/bootstrap/bootstrap.go]: Is this making HTTPS mandatory? It should be optional, selected by ENV vars

Comment (mteodor) [reply internal/pkg/bootstrap/bootstrap.go]: nope, it works with http as well

Comment (dusanb94) [reply internal/pkg/bootstrap/bootstrap.go]: Please don't ignore the error here.

File: internal/pkg/bootstrap/bootstrap.go [hunk 2]
```diff
@@ -4,6 +4,8 @@
 package bootstrap
 
 import (
+	"crypto/tls"
+	"crypto/x509"
 	"encoding/json"
 	"errors"
```
Comment (manuio) [inline internal/pkg/bootstrap/bootstrap.go]: you could replace this package by `mainflux/errors`. Also, I'm not sure that you are using the error created at line 160

File: internal/pkg/bootstrap/bootstrap.go [hunk 3]
```diff
@@ -15,6 +17,7 @@ import (
 	"github.com/mainflux/agent/internal/app/agent"
 	"github.com/mainflux/agent/internal/pkg/config"
 	export "github.com/mainflux/export/pkg/config"
+	errors "github.com/mainflux/mainflux/errors"
```
Comment (manuio) [inline internal/pkg/bootstrap/bootstrap.go]: No need to specify this alias

**Pull request merged**